bundles/at.medevit.elexis.agenda.ui/rsc/html/switchParallel.html

```diff
@@ -142,33 +142,59 @@
 
       // https://fullcalendar.io/docs/v3/resourceOrder
       
+      var colorMapping = {};
       function loadResources(callback) {
         var _resources = loadResourcesFunction();
         var resources = JSON.parse(_resources);
+          resources.forEach(function(resource) {
+            colorMapping[resource.id] = 'rgba(' + resource.color + ', 0.2)';
+          });
         callback(resources);
-      }      
+      }
 
+      function getColorForResource(resourceId) {
+        var color = colorMapping[resourceId] || 'rgba(255, 255, 255, 0.2)';
+        return color;
+      }
       function loadEventsCallback(start, end, timezone, callback) {
         var eventsJson = loadEventsFunction(start.format(), end.format(), timezone);
         var jsonEvents = JSON.parse(eventsJson);
-
         var events = [];
-        jsonEvents.forEach(function (event) {
-          events.push({
-            id: event.id,
-            title: event.title,
-            start: event.start,
-            end: event.end,
-            description: event.description,
-            icon: event.icon,
-            rendering: event.rendering,
-            borderColor: event.borderColor,
-            backgroundColor: event.backgroundColor,
-            textColor: event.textColor,
-            resourceId: event.resource,
-            allDay: event.allDay
+        var processedResources = new Set();
+          jsonEvents.forEach(function (event) {
+            var resourceId = event.resource || event.resourceId;
+              if (!processedResources.has(resourceId)) {

```

Comment:
Wenn hier ein zusätzlicher check wäre ob in dem `colorMapping` eine Farbe für die Resource hinterlegt ist, könnten nur für Resourcen für die eine Farbe Konfiguriert ist ein background event in der Farbe angelegt werden.

Dann wäre die Änderung im css nicht nötig, und es würden Resourcen ohne Farbe wie bisher dargestellt werden.

Bitte wenn es keine Einwände gibt, bitte so implementieren.